plugins/obs-webrtc/whip-output.cpp

```diff
@@ -465,31 +535,41 @@ bool WHIPOutput::Connect()
 	auto response = std::string(read_buffer);
 	response.erase(0, response.find("v=0"));
 
+	// If we are sending multiple layers assert
+	// that the remote accepted them all
+	if (videoLayerStates.size() != 1) {
+		auto layersAccepted = CountSimulcastLayers(response);
+		if (videoLayerStates.size() != layersAccepted) {
+			do_log(LOG_ERROR, "WHIP only accepted %lu layers", layersAccepted);
+			displayError(std::to_string(layersAccepted).c_str(), "Error.SimulcastLayersRejected");
```

Comment:
cool 👍

Reply:
Thanks! It is so simple, and I anticipate it will help a lot :)